src/CMakeSources.cmake

```diff
@@ -24,10 +24,24 @@ set(GGPO_LIB_SRC_NOFILTER
 )
 
 if(UNIX)
-	set(GGPO_LIB_SRC_NOFILTER
-		${GGPO_LIB_SRC_NOFILTER}
-		"lib/ggpo/platform_linux.cpp"
-	)
+	if(APPLE)
+		SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -fdeclspec")
+		set(GGPO_LIB_SRC_NOFILTER
+			${GGPO_LIB_SRC_NOFILTER}
+			"lib/ggpo/platform_unix.h"
+			"lib/ggpo/platform_unix.cpp"
+			"lib/ggpo/pevents.h"
+			"lib/ggpo/pevents.cpp"
+		)
+	else(APPLE)
+		set(GGPO_LIB_SRC_NOFILTER
+			${GGPO_LIB_SRC_NOFILTER}
+			"lib/ggpo/platform_linux.h"
+			"lib/ggpo/platform_linux.cpp"
```

Comment:
`platform_linux.*` and `platform_unix.*` are almost identical. We can probably just use `platform_unix.*` on both MacOS and Linux (potentially with an #ifdef or two).

Reply:
I totally agree.